METRICS.md

```diff
@@ -152,13 +154,87 @@ We collect data for the following events:
   * `eventAction` : bento-app-link-click
   * `eventLabel` : link identifier
 
+### Interview recruitment
+
+(This is only shown occasionally, when we're trying to recruit people to join in user research.)
+
+- When the recruitment link appears on the page
+  * `hitType` : event
+  * `eventCategory` : Recruitment
+  * `eventAction` : View
+  * `eventLabel` : Recruitment text
+
+- When the recruitment link is clicked
+  * `hitType` : event
+  * `eventCategory` : Recruitment
+  * `eventAction` : Engage
+  * `eventLabel` : Recruitment text
+
+### NPS/CSAT surveys
```

Comment:
Let's write these out since here, since this is the first time we introduce them in the doc: "Net Promoter Score (NPS)" and "Customer Satisfaction (CSAT)"